engine/api/workflow/execute_node_run.go

```diff
@@ -362,6 +370,57 @@ func addJobsToQueue(db gorp.SqlExecutor, stage *sdk.Stage, run *sdk.WorkflowNode
 	return nil
 }
 
+func getJobExecutablesGroups(db gorp.SqlExecutor, run *sdk.WorkflowNodeRun) ([]sdk.Group, error) {
```

Comment:
do not return groups, add them to the workflowNodeRun

Reply:
no groups attr on WorkflowNodeRun, see. https://github.com/ovh/cds/blob/master/sdk/workflow_run.go#L99

Reply:
ok